docs/apache-airflow/issue-triage-process.rst

```diff
@@ -0,0 +1,195 @@
+such as ``2.x``, where a bug may be targeted for the next patch release milestone
+such as ``2.x.y``.
+
+In the interest of being precise, when an issue is tagged with a milestone, it
+represents that it will be considered for that release, not that it is committed to
+a release. Once a PR is created to fix that issue and when that PR is tagged with a
+milestone, it implies that the PR is intended to released in that milestone.
+
+It is worth noting that Airflow Core and Airflow Providers are now released and
+versioned separately, so they may have a different set of milestones.
+
+
+**Transient Labels**
+
+Sometimes, there is more information needed to either understand the issue or
+to be able to reproduce the issue. Typically, this may require a response to the
+issue creator asking for more information, with the issue then being tagged with
+the label ``pending-response``.
+Also, during this stage, additional labels may be added to the issue to help
+classification and triage, such as ``reported_version`` and ``area``.
+
+Occasionally an issue may require a larger discussion among the Airflow PMC or
+the developer mailing list. This issue may then be tagged with the
+``needs:discussion`` label.
+
+Some issues may need a detailed review by one of the core committers of the project
+and this could be tagged with a ``needs:triage`` label.
+
+
+**Good First Issue**
+
+Issues which are relatively straight forward to solve, will be tagged with
+the ``GoodFirstIssue`` label.
+
+The intention here is to galvanize contributions from new and inexperienced
+contributors who are looking to contribute to the project. This has been successful
+in other open source projects and early signs are that this has been helpful in the
+Airflow project as well.
+
+Ideally, these issues only require one or two files to be changed. The intention
+here is that incremental changes to existing files are a lot easier for a new
+contributor as compared to adding something completely new.
+
+Another possibility here is to add “how to fix” in the comments of such issues, so
+that new contributors have a running start when then pick up these issues.
+
+
+**Timeliness**
+
+For the sake of quick responses, the general “soft" rule within the Airflow project
+is that if there is no assignee, anyone can take an issue to solve.
+
+However, this depends on timely resolution of the issue by the assignee. The
+expectation is as follows:
+
+* If there is no activity on the issue for 2 weeks, the assignee will be reminded about the issue and asked if they are still working on it.
+* If there is no activity even after 1 more week, the issue will be unassigned, so that someone else can pick it up and work on it.
+
+
+There is a similar process when additional information is requested from the
+issue creator. After the pending-response label has been assigned, if there is no
+further information for a period of 1 month, the issue will be automatically closed.
+
+
+
+**Invalidity**
```

Comment:
we should also add note about `won't fix`

Reply:
done